src/components/plots/piePlot.ts

```diff
@@ -11,6 +11,8 @@ export module Plots {
     private static _INNER_RADIUS_KEY = "inner-radius";
     private static _OUTER_RADIUS_KEY = "outer-radius";
     private static _SECTOR_VALUE_KEY = "sector-value";
+    private _startAngles: Utils.Map<Dataset, number[]>;
+    private _endAngles: Utils.Map<Dataset, number[]>;
```

Comment:
If only one `Dataset` is supported, couldn't these just be arrays instead of a `Map`?